package.json

```diff
@@ -34,16 +35,15 @@
     "eslint-utils": "^1.4.2",
     "ethereumjs-abi": "^0.6.7",
     "ganache-cli": "^6.4.4",
+    "ganache-time-traveler": "^1.0.5",
     "husky": "^3.0.9",
     "lerna": "^3.15.0",
     "prettier": "^1.18.2",
     "solhint": "^2.1.0",
     "truffle": "^5.0.32",
-    "ganache-time-traveler": "^1.0.5",
     "truffle-flatten": "^1.0.6",
-    "truffle-verify": "^1.0.4",
-    "@truffle/hdwallet-provider": "^1.0.17",
-    "truffle-hdwallet-provider-privkey": "^1.0.3"
+    "truffle-hdwallet-provider-privkey": "^1.0.3",
+    "truffle-verify": "^1.0.4"
   },
   "prettier": {
     "trailingComma": "es5",
```

Comment:
this file was reordered by yarn because I updated `truffle-verify` and `ganache-time-traveler`